src/Tracing/HttpClient/AbstractTraceableHttpClient.php

```diff
@@ -52,6 +52,12 @@ public function request(string $method, string $url, array $options = []): Respo
             $headers['sentry-trace'] = $parent->toTraceparent();
 
             $uri = new Uri($url);
+            $partialUri = Uri::fromParts([
+                    'scheme' => $uri->getScheme(),
+                    'host' => $uri->getHost(),
+                    'port' => $uri->getPort(),
+                    'path' => $uri->getPath(),
+                ]);
```

Comment:
CS seems off here:

```suggestion
            $partialUri = Uri::fromParts([
                  'scheme' => $uri->getScheme(),
                  'host' => $uri->getHost(),
                  'port' => $uri->getPort(),
                  'path' => $uri->getPath(),
              ]);
```

And:

`l`: The only "worry" I have here is that a `MalformedUriException` is thrown for some reason. Although this might be totally unfounded since a URI was already there but maybe we want to try/catch anyway just to be 100% sure since not all PSR-7's might be created equal.